
  ==========================================
  Studio Pro version: 6.5.4
  Started at: 15.11.2021, 15:05:11
  ==========================================
  

[2021-11-15T12:05:29.042Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:05:29.119Z] : [info]: ---------------------------------------------

[2021-11-15T12:05:29.119Z] : [success]: Бот начал работу

[2021-11-15T12:05:29.120Z] : [info]: Robot start

[2021-11-15T12:06:37.416Z] : [error]: Script Stopped.

[2021-11-15T12:06:37.419Z] : [info]: ---------------------------------------------

[2021-11-15T12:13:03.899Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:13:04.060Z] : [success]: Бот начал работу

[2021-11-15T12:13:04.061Z] : [info]: Robot start

[2021-11-15T12:13:09.137Z] : [error]: "script.exe" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.


[2021-11-15T12:13:09.139Z] : [error]: Время выполнения: 5с 75мс

[2021-11-15T12:13:09.140Z] : [info]: ---------------------------------------------

[2021-11-15T12:16:49.057Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:16:49.150Z] : [success]: Бот начал работу

[2021-11-15T12:16:49.151Z] : [info]: Robot start

[2021-11-15T12:16:53.264Z] : [error]: "script.exe" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.


[2021-11-15T12:16:53.268Z] : [error]: Время выполнения: 4с 113мс

[2021-11-15T12:16:53.270Z] : [info]: ---------------------------------------------

[2021-11-15T12:48:35.758Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:48:35.792Z] : [success]: Бот начал работу

[2021-11-15T12:48:35.793Z] : [info]: Robot start

[2021-11-15T12:48:40.154Z] : [error]: C:\Users\sugan\AppData\Local\Programs\Python\Python39\python.exe: can't open file 'C:\Program Files\ElectroNeek Studio Pro\script.py': [Errno 2] No such file or directory


[2021-11-15T12:48:40.157Z] : [error]: Время выполнения: 4с 360мс

[2021-11-15T12:48:40.158Z] : [info]: ---------------------------------------------

[2021-11-15T12:49:25.854Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:49:25.927Z] : [success]: Бот начал работу

[2021-11-15T12:49:25.928Z] : [info]: Robot start

[2021-11-15T12:49:30.135Z] : [error]: C:\Users\sugan\AppData\Local\Programs\Python\Python39\python.exe: can't open file 'D:\╨а╨░╨▒╨╛╤З╨╕╨╣': [Errno 2] No such file or directory


[2021-11-15T12:49:30.138Z] : [error]: Время выполнения: 4с 206мс

[2021-11-15T12:49:30.140Z] : [info]: ---------------------------------------------

[2021-11-15T12:49:59.167Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:49:59.189Z] : [success]: Бот начал работу

[2021-11-15T12:49:59.190Z] : [info]: Robot start

[2021-11-15T12:50:03.277Z] : [error]: Системе не удается найти указанный путь.


[2021-11-15T12:50:03.280Z] : [error]: Время выполнения: 4с 87мс

[2021-11-15T12:50:03.281Z] : [info]: ---------------------------------------------

[2021-11-15T12:50:32.977Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:50:33.017Z] : [success]: Бот начал работу

[2021-11-15T12:50:33.018Z] : [info]: Robot start

[2021-11-15T12:50:35.124Z] : [error]: Системе не удается найти указанный путь.


[2021-11-15T12:50:35.127Z] : [error]: Время выполнения: 2с 105мс

[2021-11-15T12:50:35.129Z] : [info]: ---------------------------------------------

[2021-11-15T12:52:31.388Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:52:31.414Z] : [success]: Бот начал работу

[2021-11-15T12:52:31.415Z] : [info]: Robot start

[2021-11-15T12:52:35.120Z] : [error]: Traceback (most recent call last):
  File "D:\Projects\Work\Scripts\script.py", line 1, in <module>
    with open("hello.txt", "w") as file:
PermissionError: [Errno 13] Permission denied: 'hello.txt'


[2021-11-15T12:52:35.123Z] : [error]: Время выполнения: 3с 705мс

[2021-11-15T12:52:35.124Z] : [info]: ---------------------------------------------

[2021-11-15T12:53:35.854Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:53:35.881Z] : [success]: Бот начал работу

[2021-11-15T12:53:35.882Z] : [info]: Robot start

[2021-11-15T12:53:40.158Z] : [error]: Traceback (most recent call last):
  File "D:\Projects\Work\Scripts\script.py", line 1, in <module>
    with open("hello.txt", "w") as file:
PermissionError: [Errno 13] Permission denied: 'hello.txt'


[2021-11-15T12:53:40.161Z] : [error]: Время выполнения: 4с 275мс

[2021-11-15T12:53:40.162Z] : [info]: ---------------------------------------------

[2021-11-15T12:55:11.720Z] : [success]: Бот начал работу

[2021-11-15T12:55:11.722Z] : [info]: Robot start

[2021-11-15T12:55:15.131Z] : [error]: Traceback (most recent call last):
  File "D:\Projects\Work\Scripts\script.py", line 1, in <module>
    with open("hello.txt", "w") as file:
PermissionError: [Errno 13] Permission denied: 'hello.txt'


[2021-11-15T12:55:15.134Z] : [error]: Время выполнения: 3с 409мс

[2021-11-15T12:55:15.135Z] : [info]: ---------------------------------------------

[2021-11-15T12:56:24.240Z] : [success]: Бот начал работу

[2021-11-15T12:56:24.241Z] : [info]: Robot start

[2021-11-15T12:56:26.853Z] : [info]: 213

[2021-11-15T12:56:28.138Z] : [success]: Бот успешно завершил работу, время выполнения: 3с 894мс

[2021-11-15T12:56:28.140Z] : [info]: ---------------------------------------------

[2021-11-15T12:56:42.419Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:56:42.437Z] : [success]: Бот начал работу

[2021-11-15T12:56:42.437Z] : [info]: Robot start

[2021-11-15T12:56:44.549Z] : [info]: 

[2021-11-15T12:56:46.132Z] : [success]: Бот успешно завершил работу, время выполнения: 3с 693мс

[2021-11-15T12:56:46.134Z] : [info]: ---------------------------------------------


  ==========================================
  Studio Pro version: 6.5.4
  Started at: 15.11.2021, 15:58:33
  ==========================================
  

[2021-11-15T12:59:16.928Z] : [success]: Workflow успешно сохранен

[2021-11-15T12:59:16.955Z] : [success]: Бот начал работу

[2021-11-15T12:59:16.956Z] : [info]: Robot start

[2021-11-15T12:59:21.123Z] : [error]: Error saving script

[2021-11-15T12:59:21.299Z] : [error]: "D:\Рабочий" не является внутренней или внешней
командой, исполняемой программой или пакетным файлом.


[2021-11-15T12:59:21.302Z] : [error]: Время выполнения: 4с 343мс

[2021-11-15T12:59:21.303Z] : [info]: ---------------------------------------------

